### ISSUE TITLE
some ideas about things to include?

### ISSUE THREAD

**Issue by aparrish** [cvalenzuela/hpc#1]
Hi, I'm going through this tutorial now and it's very helpful. I'm finding some things on the HPC wiki that might be nice to include in some way eventually:

* [How to copy files](https://wikis.nyu.edu/pages/viewpage.action?pageId=84612833)
* [Setting up an SSH tunnel](https://wikis.nyu.edu/pages/viewpage.action?pageId=84607390)
* [Running Jupyter on Prince](https://wikis.nyu.edu/display/NYUHPC/Running+Jupyter+on+Prince)

Thoughts?

**Comment by cvalenzuela** [comment]
yes! we should definitely include those. Feel free to update the readme.

For copying files, I think you can transfer with a regular `scp` and there also [this tool](https://wikis.nyu.edu/display/NYUHPC/Using+Globus+to+transfer+files+to+and+from+NYU+HPC+storage). maybe we should point to both?

**Comment by cvalenzuela** [comment]
I just updated the `README.md` with how to copy files and setting up an SHH tunnel.
@aparrish would you add anything else?
